test/integration/package_int_test.go

```diff
@@ -755,6 +755,37 @@ func (suite *PackageIntegrationTestSuite) TestCVE_Indirect() {
 	cp.ExpectExitCode(1)
 }
 
+func (suite *PackageIntegrationTestSuite) TestChangeSummary() {
+	suite.OnlyRunForTags(tagsuite.Package)
+	ts := e2e.New(suite.T(), false)
+	defer ts.Close()
+
+	cp := ts.Spawn("config", "set", constants.AsyncRuntimeConfig, "true")
+	cp.Expect("Successfully set")
+	cp.ExpectExitCode(0)
+
+	cp = ts.SpawnWithOpts(
+		e2e.OptArgs("checkout", "ActiveState-CLI/small-python", "."),
+		e2e.OptAppendEnv(constants.DisableRuntime+"=false"),
+	)
+	cp.Expect("Checked out", e2e.RuntimeSourcingTimeoutOpt)
+	cp.ExpectExitCode(0)
+
+	cp = ts.SpawnWithOpts(
+		e2e.OptArgs("install", "requests@2.31.0"),
```

Comment:
Perhaps worth pinning the timestamp here also so this doesn't break with inventory changes.

Reply:
I tried this, but for whatever reason, the command just immediately prints "Added package: ".

Reply:
Did you file a bug?

Reply:
DX-2864